kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/embedder/MavenProjectLoader.java

```diff
@@ -88,24 +89,32 @@ public static MavenProject parseMavenPom(InputStream pomStream, boolean offline)
     public static MavenEmbedder newMavenEmbedder(boolean offline) {
         MavenRequest mavenRequest = createMavenRequest(offline);
         MavenEmbedder mavenEmbedder;
-            try {
-                mavenEmbedder = new MavenEmbedder( mavenRequest );
-            } catch (MavenEmbedderException e) {
-                log.error("Unable to create new MavenEmbedder", e);
-                throw new RuntimeException(e);
-            }
+        try {
+            mavenEmbedder = new MavenEmbedder( mavenRequest );
+        } catch (MavenEmbedderException e) {
+            log.error("Unable to create new MavenEmbedder", e);
+            throw new RuntimeException(e);
+        }
         return mavenEmbedder;
     }
 
+    public static String getGlobalRepo(){
+        String global = System.getProperty(GLOBAL_M2_REPO_URL);
+        if(global == null){
+            global = "repositories" +File.separator +"kie" +File.separator +"global";
```

Comment:
this is for now acceptable in order to avoid a circular dependency with AppFormer (where the env variable is defined and properly default`ed); this is a temporary solution and will be replaced anyway by the new maven build

Reply:
Refactored and commented